source/docs/power-and-electronics/servo-guide/usage-tips.rst

```diff
@@ -39,17 +28,35 @@ There are also some alternative designs of servo supports; one of them, which is
 
      Linkage example, courtesy of team 4137 Islandbots. A goBILDA flat beam is used as the link.
 
-- If you need more power, use a `REV Servo Power Module <https://www.revrobotics.com/rev-11-1144/>`_.
+- If you need more power, use a `REV Servo Hub <https://www.revrobotics.com/rev-11-1855/>`_ or `goBILDA 6V Servo Power Injector <https://www.gobilda.com/6v-servo-power-injector-6-channel-8-15v-input/>`_. **This is important for servos under high load conditions, such as the Axon servos.**
```

Comment:
I would probably nix the axon reference too, servos under high load period should get a SPI